tasks/eva_3655/remediateLowerCaseNucleotide.py

```diff
@@ -0,0 +1,223 @@
+import argparse
+import hashlib
+import logging
+import os
+
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+from ebi_eva_internal_pyutils.mongo_utils import get_mongo_connection_handle
+
+logger = log_cfg.get_logger(__name__)
+log_cfg.add_stdout_handler()
+log_cfg.set_log_level(logging.INFO)
+
+# regex_pattern = r'^[a-zA-Z0-9_.]+_[0-9]+_([a-zA-Z]*[^ACGTN]+[a-zA-Z]*_?[a-zA-Z]*|[a-zA-Z]*_?[a-zA-Z]*[^ACGTN]+[a-zA-Z]*)$'
+regex_pattern = r'^[a-zA-Z0-9_.]+_[0-9]+_([a-zA-Z]*[a-z]+[a-zA-Z]*_?[a-zA-Z]*|[a-zA-Z]*_?[a-zA-Z]*[a-z]+[a-zA-Z]*)$'
```

Comment:
Just trying to understand the regex
```suggestion
regex_pattern = r'^[a-zA-Z0-9_.]+_[0-9]+_([a-zA-Z]*[a-z]+[a-zA-Z]*_?[a-zA-Z]*|[a-zA-Z]*_?[a-zA-Z]*[a-z]+[a-zA-Z]*)$'
#                   (seq name)   _(pos) _ (Ref with lower case _ alt without)or(Ref without _ alt with lower case)
```
Why are the underscore between REF and ALT optional ?